### PR TITLE
Tdl 9721 missing top level breadcrumb for all streams

### PR DESCRIPTION
**Pull request by dbshah1212**
# Description of change
- Added top-level breadcrumb

# Manual QA steps
 - Run discover mode with and without this code and verify fields of both catalogs. 
 - There must be no missing fields.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_bing_ads/__init__.py [hunk 1]
```diff
@@ -299,30 +299,38 @@ def get_type_map(client):
...
+                key_properties = pks,
+                valid_replication_keys = replication_key,
+                replication_method = 'INCREMENTAL' if replication_key else 'FULL_TABLE'
+            )
+        )
+
+    # Marking replication key as automatic
+    for field_name in schema['properties'].keys():
+        if field_name == replication_key:
+            mdata = metadata.write(mdata, ('properties', field_name), 'inclusion', 'automatic')
```
Comment (savan-chovatiya) [inline tap_bing_ads/__init__.py:326]:
```suggestion
    if replication_key:
        mdata = metadata.write(mdata, ('properties', replication_key), 'inclusion', 'automatic')
```

Comment (dbshah1212) [reply tap_bing_ads/__init__.py]: Updated

**Review by karanpanchal-crest — APPROVED** [review]

**Comment by karanpanchal-crest** [comment]
@dbshah1212 Why is the CircleCI build failing for this PR?

**Review by KrisPersonal — APPROVED** [review]

**Review by kspeer825 — CHANGES_REQUESTED** [review]
I believe more of the standard assertions can for discovery can be applied in the tap-tester test now. Per this comment https://jira.talendforge.org/browse/TDL-9721?focusedCommentId=1003711&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1003711 it looks like replication key and primary keys are now properly set in metadata but the test is only verifying against primary keys.

**Comment by dbshah1212** [comment]
> I believe more of the standard assertions can for discovery can be applied in the tap-tester test now. Per this comment https://jira.talendforge.org/browse/TDL-9721?focusedCommentId=1003711&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1003711 it looks like replication key and primary keys are now properly set in metadata but the test is only verifying against primary keys.

Yes updated the replication keys in the base file as there is already a discrepancy in primary keys and replication keys and we already created a card for the same: https://jira.talendforge.org/browse/TDL-15816. And uncommented replication key test and it's passing now.

**Review by kspeer825 — APPROVED** [review]

**Pull request merged**